src/codewhisperer/service/securityIssueProvider.ts

```diff
@@ -17,6 +16,9 @@ export abstract class SecurityIssueProvider {
     }
 
     public handleDocumentChange(event: vscode.TextDocumentChangeEvent) {
+        if (!event.contentChanges || event.contentChanges.length === 0) {
```

Comment:
please add a code comment explaining why this is needed, e.g.:

> `handleDocumentChange` is triggered because of change in document due to our own code generation.